ui/statistics/src/main/java/de/danoeh/antennapod/ui/statistics/years/BarChartView.java

```diff
@@ -0,0 +1,138 @@
+package de.danoeh.antennapod.ui.statistics.years;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.ColorFilter;
+import android.graphics.DashPathEffect;
+import android.graphics.Paint;
+import android.graphics.PixelFormat;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import androidx.annotation.NonNull;
+import androidx.appcompat.widget.AppCompatImageView;
+import de.danoeh.antennapod.ui.common.ThemeUtils;
+import de.danoeh.antennapod.ui.statistics.StatisticsColorScheme;
+import io.reactivex.annotations.Nullable;
```

Comment:
This should be replaced with the AndroidX Annotation Nullable. I'm assuming this one was just chosen automatically by the IDE's auto-import.